routes/project.js

```diff
@@ -219,6 +219,71 @@ module.exports = function (express) {
           });
         });
       },
+      function(callback) {
+        // Get list of templates
+        var templateDir = path.join(process.cwd(), data.format);
+
+        fse.ensureDir(templateDir, function(error) {
+          if (error) {
+            return callback(error);
+          }
+
+          var items = require(path.join(templateDir, 'list.json'));
+          callback(null, items);
+        });
+      },
+      function (templateList, callback) {
+        var count = 0,
+          includes = [],
+          numberOfIncludes,
+          template;
+
+        template = templateList.filter(function (template) {
+          return template.src === data.templateName;
+        });
+        if (!template) { // template not found on list
+          return callback(null);
+        }
+        template = template[0];
+
+        numberOfIncludes = (!template.includes) ? 0 : template.includes.length;
+
+        // Skip copy template when empty check box is enabled
+        if (numberOfIncludes === 0) {
+          return callback(null);
+        }
+
+        function onLoadSuccess() {
+          count++;
+          if (count === numberOfIncludes) {
+            callback(null);
+          }
+        }
+
+        // Copy the includes to project folder
+        fse.ensureDir(projectPath, function (error) {
+          if (error) {
+            return callback(error);
+          }
+
+          template.includes.forEach(function (include) {
+            var dest = path.join(projectPath, include.dest),
+              options = {
```

Comment:
Are those options needed?

Reply:
Values of options are needed, but variable "options" my be omitted. I wanted to remain the same style for all "copy" method parameters. I mean "src", "dest", "options".
Instead of above I can write also: `fse.copy(include.src, path.join(projectPath, include.dest), {"overwrite"....}, function (err)` but is this better readable?